libs/application/template-api-modules/src/lib/modules/templates/driving-license-submission/emailGenerators/messages.ts

```diff
@@ -2,17 +2,20 @@ export const m = {
   drivingLicenseSubject: {
     'B-temp': 'Umsókn um ökunám móttekin',
     'B-full': 'Umsókn um fullnaðarskírteini móttekin',
+    BE: 'Umsókn um kerruréttindi móttekin',
   },
   drivingLicenseHeading: {
     'B-temp': 'Umsókn þín um ökunám hefur verið móttekin',
     'B-full': 'Umsókn þín um fullnaðarskírteinið hefur verið móttekin',
+    BE: 'Umsókn um kerruréttindi móttekin',
   },
   inPersonRequirements: {
     title: {
       'B-temp':
         'Til að klára umsóknina þarf að koma á skrifstofu sýslumanns og skila eftirfarandi gögnum',
       'B-full':
         'Áður en hægt er að panta fullnaðarskírteini, þarf að koma á skrifstofu sýslumanns og skila eftirfarandi gögnum',
+      BE: 'Áður en hægt er að panta fullnaðarskírteini, þarf að koma á skrifstofu sýslumanns og skila eftirfarandi gögnum',
```

Comment:
Should this text be the same here as in `B-full`?

Reply:
no, fixed